src/kvstore/Part.cpp

```diff
@@ -45,13 +38,14 @@ Part::Part(GraphSpaceID spaceId,
            const std::string& walPath,
            KVEngine* engine,
            std::shared_ptr<folly::IOThreadPoolExecutor> ioPool,
-           std::shared_ptr<thread::GenericThreadPool> workers)
+           std::shared_ptr<thread::GenericThreadPool> workers,
+           wal::BufferFlusher* flusher)
         : RaftPart(FLAGS_cluster_id,
```

Comment:
is it possible that a part belongs to two clusters?
what does this cluster_id used for?

Reply:
Please refer #526